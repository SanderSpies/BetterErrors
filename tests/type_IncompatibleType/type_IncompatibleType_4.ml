module TopNotchModule = struct
  type jeSuisString = string
  type woShiRecordIHaveAReallyLongNameThatllMakeYouLineBreak = {nihao: int}
end

let asd: TopNotchModule.jeSuisString = ({nihao = 1}: TopNotchModule.woShiRecordIHaveAReallyLongNameThatllMakeYouLineBreak)
