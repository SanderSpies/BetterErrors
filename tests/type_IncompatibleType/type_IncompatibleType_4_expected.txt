[36mtests/type_IncompatibleType/type_IncompatibleType_4.ml:6 40-51
[0m3 |   type woShiRecordIHaveAReallyLongNameThatllMakeYouLineBreak = {nihao: int}
4 | end
5 | 
6 | let asd: TopNotchModule.jeSuisString = ([31;4m{nihao = 1}[0m: TopNotchModule.woShiRecordIHaveAReallyLongNameThatllMakeYouLineBreak)

Warning 40: this record of type TopNotchModule.woShiRecordIHaveAReallyLongNameThatllMakeYouLineBreak contains fields that are 
not visible in the current scope: nihao.
They will not be selected if the type becomes unknown.
[36mtests/type_IncompatibleType/type_IncompatibleType_4.ml:6 40-51
[0m3 |   type woShiRecordIHaveAReallyLongNameThatllMakeYouLineBreak = {nihao: int}
4 | end
5 | 
6 | let asd: TopNotchModule.jeSuisString = ([31;4m{nihao = 1}[0m: TopNotchModule.woShiRecordIHaveAReallyLongNameThatllMakeYouLineBreak)

[31;4mThis is:[0m  TopNotchModule.woShiRecordIHaveAReallyLongNameThatllMakeYouLineBreak
[32mWanted:[0m   TopNotchModule.jeSuisString = string                                
